test/test_site.rb

```diff
@@ -14,14 +14,20 @@ class TestSite < JekyllUnitTest
 
     should "have an array for plugins if passed as a string" do
       site = Site.new(site_configuration({ "plugins_dir" => "/tmp/plugins" }))
-      assert_equal ["/tmp/plugins"], site.plugins
+      array = Utils::Platforms.really_windows? ? ["C:/tmp/plugins"] : ["/tmp/plugins"]
```

Comment:
I'm just saying, it's bad practice hardcoding paths. Especially when you can use something a lot more reliable to detect the path.